examples/tutorials/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,1181 @@
+{
```

Comment:
<p>we don't see the loss reducing, why is so?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>yes this the reason for the low acc in the last discussion you started I guess. Re-running the training, but still I don't understand why sometimes things like that occur, this is why I maybe suggested an issue with GarbageCollector in my notebook (because usually I don't get these kind of error if I restart the notebook and directly train only one model)</p>

Reply:
<p>Ok I ran it again and the loss was 715 and 715 for both epochs, still the test-acc. was 0.14 which should be correct, so I guess the optim together with the quite small batch size just leads to bad first steps. </p>